Makefile.usr

```diff
@@ -302,6 +302,8 @@ SDLCFLAGS = $(shell sdl-config --cflags)
 SDLLIBS = $(shell sdl-config --libs)
 FREETYPECFLAGS = $(shell freetype-config --cflags)
 FREETYPELIBS = $(shell freetype-config --libs)
+NCURSESCFLAGS = $(shell ncurses6-config --cflags)
+NCURSESLIBS = $(shell ncurses6-config --libs)
```

Comment:
@chewi I am not sure what this file is for, but it seems like it should have some `SLANGCFLAGS` and `SLANGLIBS` definitions somewhere?  I also didn't make an attempt to add anything for Windows, Dos or any other targets outside of Linux as I am not sure what the impact maybe.  Just let me know if any adjustments are need.

Reply:
I think this is only used for DOS and Windows. It's merged now so hey ho.